### ISSUE TITLE
Add limit to event retrieval endpoint

### ISSUE THREAD

**Issue by johannesduesing** [delphi-hub/delphi-registry#127]
**Describe the problem**
When a system is running for multiple days or weeks, the event list of instances may become very long. The Management development team would like to have an option to limit the number of events returned by the ```GET /instances/{Id}/events``` endpoint, which would make displaying them in single pages easier.

**Describe the solution you'd like**
Allow users to limit the maximum number of  events returned by the endpoint. In my opinion this should be a query parameter named something like ```LimitItems``` of type ```int```. Also think of having an option to pass the number of events the user already *has*, and return the *next* ones. May look like this
```
GET /instances/{id}/events?StartPage=3&PageItems=20&LimitItems=100
```
This would request the next 100 events, but also indicate that the caller already has 3 pages a 20 items, meaning that the registry should skip the first 20*3 = 90 items.

**Comment by sami-cseseu** [comment]
The route will be like
`GET /instances/{id}/eventList?StartPage=3&PageItems=20&LimitItems=100`

**Comment by johannesduesing** [comment]
Closed with #128
